.clang-format

```diff
@@ -0,0 +1,5 @@
+﻿---
+BasedOnStyle: Google
+IndentWidth: '4'
+SortIncludes: false
```

Comment:
Is there a reason we don’t want to sort includes?

Reply:
I didn't because it was annoying me and I wasn't a fan of how it was sorting them.
I usually sort includes in other from most specific to most general.
 Probably better to sort them with clang though :/